velox/dwio/parquet/writer/Writer.h

```diff
@@ -109,6 +110,9 @@ struct WriterOptions : public dwio::common::WriterOptions {
   std::optional<std::string> parquetWriteTimestampTimeZone;
   bool writeInt96AsTimestamp = false;
   std::optional<bool> useParquetDataPageV2;
+  std::optional<bool> storeDecimalAsInteger;
```

Comment:
We don't have to change the physical type and the ParquetVersion.
Java can use FLBA, Velox can use integers (more efficient).
We only need to ensure the Decimal logical type is the same as specified by the client (Presto in this case).